J/jemalloc/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "jemalloc"
+version = v"5.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/jemalloc/jemalloc.git", "886e40bb339ec1358a5ff2a52fdb782ca66461cb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd jemalloc/
+autoconf
+
+if [[ "${target}" == "x86_64-linux-gnu" ]]; then
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-initial-exec-tls
+else
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+fi
+
+make
+make install
+
+# Rename .dll for Windows targets.
+if [[ "${target}" == *"w64"* ]]; then
+    mkdir -p ${libdir}
+    mv ${prefix}/lib/jemalloc.dll ${libdir}/libjemalloc.dll
```

Comment:
Also why moving this?  Doesn't the audit move the file automatically to `${libdir}`?

Reply:
You should _not_ change the name of libraries: downstream libraries which expect the original name won't like your intervention.

Instead, [`LibraryProduct`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.LibraryProduct) allows specifying multiple names exactly to cater for platform-dependent names.  And you can see what was done in #932:
https://github.com/JuliaPackaging/Yggdrasil/blob/8c11cdbee94fe05deaee977b4785e87bc2742747/J/jemalloc/build_tarballs.jl#L31

Reply:
Thanks, will incorporate! (I wonder if it might make sense to add a Ygg PR check to the BinaryBuilder wizard to alert ppl to duplicate PRs?)

Reply:
https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/1099